### PR TITLE
ES-48266: fix date crash for language except english

### PR DESCRIPTION
**Pull request by nirajrajgor**
ES-48266
DatePicker crashed for values other than english. Add a fallback as Moment object to prevent crash.

Dashboard PR:
https://github.com/locus-taxy/locus-dashboard-v2/pull/5113

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -198,7 +198,8 @@ class Kronos extends Component {
   }
 
   save(saving) {
-    const { datetime } = this.state
+    let { datetime } = this.state
```
Comment (harikrishnakarur) [inline src/index.js:201]: Is the dateTime null or undefined? If its undefined we can keep it cont with fallback as Moment()
```suggestion
    const { datetime = Moment() } = this.state
```

Comment (nirajrajgor) [reply src/index.js:201]: It's null. So default value will not work.

**Comment by nirajrajgor** [comment]
@Mayank1791989 Please have a look at PR.

File: src/index.js [hunk 2]
```diff
@@ -301,7 +302,7 @@ class Kronos extends Component {
       this.toggle(false)
       if (this.props.onBlur) this.props.onBlur(e)
     }
-    if (this.state.input == this.state.datetime.format(this.format())) {
+    if (this.state.input == datetime.format(this.format())) {
```
Comment (harikrishnakarur) [inline src/index.js:305]:
```suggestion
    if (this.state.input == this.state.datetime?.format(this.format())) {
```

Why not just null check?

Comment (nirajrajgor) [reply src/index.js:305]: Null check will also work, but they already initialized the `datetime` variable at the top and did not use it so i thought this is the right place to use.

**Review by Mayank1791989 — COMMENTED** [review]
We use changeset for auto release, check their docs we have to add a changset in the PR. https://github.com/changesets/changesets

**Comment by nirajrajgor** [comment]
Not seeing the errors in the bugsnag anymore. Hence closing the PR.

**Pull request closed without merging**